### PR TITLE
chore: improve dbt-athena profile page

### PR DESCRIPTION
**Pull request by nicor88**
## What are you changing in this pull request and why?

Add some more relevant parameters to work with dbt-athena that were missing in the official dbt page.

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [x] Add/remove page in `website/sidebars.js`
- [x] Provide a unique filename for new pages
- [x] Add an entry for deleted pages in `website/vercel.json`
- [x] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/docs/core/connect-data-platform/athena-setup.md [hunk 1]
```diff
@@ -33,12 +33,14 @@ default:
...
       s3_staging_dir: [s3_staging_dir]
+      s3_data_dir: [s3_data_dir]
+      s3_data_naming: [table_unique] # the type of naming convention used when writing to S3
       region_name: [region_name]
       database: [database name]
       schema: [dev_schema]
-      aws_profile_name:
-        [optional, profile to use from your AWS shared credentials file.]
-
+      aws_profile_name: [optional, profile to use from your AWS shared credentials file.]
```
Comment (mirnawong1) [inline website/docs/docs/core/connect-data-platform/athena-setup.md]:
```suggestion
      aws_profile_name: [optional profile to use from your AWS shared credentials file.]
```

File: website/docs/docs/core/connect-data-platform/athena-setup.md [hunk 2]
```diff
@@ -33,12 +33,14 @@ default:
...
+      s3_data_naming: [table_unique] # the type of naming convention used when writing to S3
       region_name: [region_name]
       database: [database name]
       schema: [dev_schema]
-      aws_profile_name:
-        [optional, profile to use from your AWS shared credentials file.]
-
+      aws_profile_name: [optional profile to use from your AWS shared credentials file.]
+      threads: [1 or more]
+      num_retries: [0 or more] # number of retries performed by the adapter, by default 5
```
Comment (mirnawong1) [inline website/docs/docs/core/connect-data-platform/athena-setup.md]:
```suggestion
      num_retries: [0 or more] # number of retries performed by the adapter. Defaults to 5
```

**Review by mirnawong1 — APPROVED** [review]
hey @nicor88 ! thanks for opening this up! will get this merged for you ✨!

**Comment by nicor88** [comment]
@mirnawong1 thanks 💯 , I totally missed your suggestion and seen them only now.

**Pull request merged**